content/documentation/pages/5-feature-guides/2-batch/4-monitoring.md

```diff
@@ -89,7 +89,13 @@ dataflow:>task launch --name task1
 dataflow:>task launch --name task2
 ```
 
-In the [DataFlow task execution UI](http://localhost:9393/dashboard/#/tasks/executions).
+In the [DataFlow task execution UI](http://localhost:9393/dashboard/#/tasks/executions) you should see:
```

Comment:
The dashboard points to localhost. Do we want that?

Reply:
Yes because this is the local platform docker-compose will expose the scdf server at localhost.